### ISSUE TITLE
OpenAPI 3.0 does not validate with a boolean response type

### ISSUE THREAD

**Issue by kentmaxwell** [a85/POSTMan-Chrome-Extension#8474]
**Describe the bug**
When creating an OpenAPI 3.0 API schema, it is not possible to use a boolean data type.  Consider this simple API definition:  https://pastebin.com/HqPrfeXA

If you try to create a collection against this schema and then validate it you will receive this error:
```The response body needs to be of type boolean, but we found a boolean instead.```

**Expected behavior**
The schema should validate just fine against the generated collection -- no error.

**Screenshots**
![image](https://user-images.githubusercontent.com/5932455/81758171-a56af480-9486-11ea-9615-87ad4fa9ddf5.png)

**App information (please complete the following information):**
- Desktop App
- Postman Version 7.24.0
- macOS v10.15.4

**Comment by VShingala** [comment]
@kentmaxwell Thanks for reporting this issue. I was able to reproduce this issue. But I wanted to ask a follow up question.

As defined response in your example has media-type of `application/json`, doesn't that mean that schema should be either of type `array` or `object`. Currently we are handling only this two type as valid type for schema under `application/json`.

You could try to validate similar schema mentioned below, and it should work.
```
openapi: 3.0.0
info:
  version: 1.0
  title: Demo API
paths:
  /api:
    get:
      summary: 'Demo invalid type'
      responses:
        '200':
          description: 'Cannot use type boolean'
          content:
            application/json:
                schema:
                    type: object
                    properties:
                      hello: boolean
                examples:
                    Boolean Response:
                        value:
                            false
```

**Comment by kentmaxwell** [comment]
Hello @VShingala --

Thanks for the quick follow-up.  I am new to the Open API standard, so I was bound to encounter strange things.  After reading what you wrote, it does make sense that any response using ```application/json``` would either be an ```array``` or ```object```.  In fact this realization helps me out with some other validation problems I have had in Postman.  However, it seems that other API editors using Open API allow the exact structure I wrote above and it is unclear that the Open API specification prohibits it.  

In my opinion, I think the approach implemented in Postman is the correct approach, but the error message should be more clear.  When Postman tries to validate a schema that does not use a ```array``` or ```object``` it should provide a message that provides the same understanding as what you wrote.  I have found two scenarios:

1.  Where it does something that doesn't make any sense like what I pointed out in this bug report
2.  Where the validation failed and it is impossible to see a validation report -- I don't know why.

As a new person to Open API it can be very confusing to understand what I am supposed to do to fix the situation.  

Thank you!

**Comment by VShingala** [comment]
@kentmaxwell Yeah, current message is very obscure and doesn't provide enough information to work with. However we are currently working on providing better validation experience with actionable messages in upcoming releases, which will fix this.

**Comment by abhijitkane** [comment]
@VShingala JSON supports direct values as well. The pastebin link works as expected in https://editor.swagger.io/.

**Comment by VShingala** [comment]
@kentmaxwell This issue should be fixed with the latest App release v7.33. The expected behavior mentioned by you is followed. Closing this issue for now, feel free to reopen this issue for any further problems.
